site/docs/spark-queries.md

```diff
@@ -191,57 +192,166 @@ join prod.db.table.snapshots s
+|      0|s3:.../dt=20210103/xxx.parquet|    PARQUET|{20210103}|          14|              2444|{1 -> 94, 2 -> 17}|{1 -> 14, 2 -> 14}|  {1 -> 0, 2 -> 0}|              {}|{1 -> 1, 2 -> 20210103}|{1 -> 3, 2 -> 20210103}|        null|          [4]|        null|            0|
+|      0|s3:.../dt=20210104/xxx.parquet|    PARQUET|{20210104}|          14|              2444|{1 -> 94, 2 -> 17}|{1 -> 14, 2 -> 14}|  {1 -> 0, 2 -> 0}|              {}|{1 -> 1, 2 -> 20210104}|{1 -> 3, 2 -> 20210104}|        null|          [4]|        null|            0|
+</div>
+
+#### All Manifests
+
+To show a table's valid file manifests and each file's metadata, run:
+
+```sql
+SELECT * FROM prod.db.table.all_manifests
+```
+
+<div class="markdown-table-container" markdown="block">
+| path | length | partition_spec_id | added_snapshot_id | added_data_files_count | existing_data_files_count | deleted_data_files_count| partition_summaries|
+| -- | -- | -- | -- | -- | -- | -- | -- |
+| s3:/.../table/data/dt=20210102/xxx.parquet | 6376 | 0 | 6272782676904868561 | 2 | 0 | 0 |[{false, false, 10, 11}, {false, false, 20210101, 20210101}]|
+</div>
+
+!!! Note
+    1. A valid manifest file is one that is referenced from any snapshot currently tracked by the table.
+    2. Fields within `partition_summaries` column of the manifests table correspond to `field_summary` structs within [manifest list](./spec.md#manifest-lists), with the following order:
+        - `contains_null`
+        - `contains_nan`
+        - `lower_bound`
+        - `upper_bound`
+    3. `contains_nan` could return null, which indicates that this information is not available from files' metadata.
+        This usually occurs when reading from V1 table, where `contains_nan` is not populated.
+
+### Advanced Usage
+
+#### Entries
+
+To show a table's current manifest entries as rows, for both delete and data files, run:
+
+```sql
+SELECT * FROM prod.db.table.entries
+```
+
+<div class="markdown-table-container" markdown="block">
+| status|snapshot_id | sequence_number | data_file |
+| -- | -- | -- | -- |
+|0     |7462238160765527919|0              |{0, s3://.../dt=20211001/age=10/00000-0-38e4b886-b445-40a9-8db0-58653a331aba-00001.parquet, PARQUET, {20211001, 10}, 1, 1132, {1 -> 47, 2 -> 52, 3 -> 47, 4 -> 55}, {1 -> 1, 2 -> 1, 3 -> 1, 4 -> 1}, {1 -> 0, 2 -> 0, 3 -> 0, 4 -> 0}, {}, {1 ->    , 2 -> data1, 3 -> \n   , 4 -> 20211001}, {1 ->    , 2 -> data1, 3 -> \n   , 4 -> 20211001}, null, [4], null, 0}|
+|2     |7462238160765527919|0              |{0, s3://.../dt=20211002/age=11/00001-1-dd95f618-4738-41a4-ba0e-21bbf964d0cc-00001.parquet, PARQUET, {20211002, 11}, 1, 1132, {1 -> 47, 2 -> 52, 3 -> 47, 4 -> 55}, {1 -> 1, 2 -> 1, 3 -> 1, 4 -> 1}, {1 -> 0, 2 -> 0, 3 -> 0, 4 -> 0}, {}, {1 ->    , 2 -> data2, 3 -> \v   , 4 -> 20211002}, {1 ->    , 2 -> data2, 3 -> \v   , 4 -> 20211002}, null, [4], null, 0}|
+</div>
+
+!!! Note
```

Comment:
Documentation shouldn't be inside `Note` callouts. Instead, these should be normal paragraphs and lists.